media-types.md

```diff
@@ -38,25 +38,40 @@ This section shows where the OCI Image Specification is compatible with formats
 
 **Similar/related schema**
 
-- [application/vnd.docker.distribution.manifest.list.v2+json](https://github.com/docker/distribution/blob/master/docs/spec/manifest-v2-2.md#manifest-list) - mediaType is different
+- [application/vnd.docker.distribution.manifest.list.v2+json](https://github.com/docker/distribution/blob/master/docs/spec/manifest-v2-2.md#manifest-list)
```

Comment:
Do you want to point to a tag here too (instead of master)?

Reply:
"and is _reserved_ in OCI"?

Reply:
> Do you want to point to a tag here too (instead of master)?

Not in this PR. There is no tag for the latest revision of the file.
https://github.com/distribution/distribution/commit/c432849e44f960c95937e43cff2d010187f7cc0b

Reply:
Added "and is reserved in OCI"?